apps/backend/src/websocket/server.py

```
@@ -0,0 +1,246 @@
+                                    "topics": subscription_msg.topics,
+                                }
+                            )
+                        )
+                    except Exception as e:
+                        logger.debug(
+                            f"Failed to send subscription confirmation to {client_id}: {e}"
+                        )
+                        break
+
+                elif message_type == MessageType.HEARTBEAT:
+                    heartbeat_msg = HeartbeatMessage(**message_data)
```

Ruff (0.12.2)

Local variable `heartbeat_msg` is assigned to but never used

Remove assignment to unused variable `heartbeat_msg`

(F841)